rskj-core/src/main/java/org/ethereum/config/SystemProperties.java

```diff
@@ -725,6 +721,16 @@ public String netName() {
         return configFromFiles.hasPath("blockchain.config.name") ? configFromFiles.getString("blockchain.config.name") : null;
     }
 
+    public boolean isRpcEnabled() {
+        return configFromFiles.hasPath(PROPERTY_RPC_ENABLED) ?
+                configFromFiles.getBoolean(PROPERTY_RPC_ENABLED) : false;
+    }
+
+    public int rpcPort() {
+        return configFromFiles.hasPath(PROPERTY_RPC_PORT) ?
+                configFromFiles.getInt(PROPERTY_RPC_PORT) : 4444;
```

Comment:
move 4444 to a variable. I would change it to a port not associated with malware if possible.

Reply:
:+1: Const created, we should discuss the default port number with other team members